src/test/pybind/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
-add_ceph_test(test_ceph_daemon.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_daemon.py)
-add_ceph_test(test_ceph_argparse.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_argparse.py)
+if(NOT FREEBSD)
+  add_ceph_test(test_ceph_daemon.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_daemon.py)
+  add_ceph_test(test_ceph_argparse.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_argparse.py)
```

Comment:
@tchaikov 
Got to dig that out again, but it has probably mostly to do with my inexperience (non existent) to setup nosetests...